python-lib/lal/classifiers/ner/text_classifier.py

```diff
@@ -26,6 +26,7 @@ def __init__(self, initial_df, queries_df, config=None):
         self.use_tokenization = True
         self.tokenizer = self.use_tokenization and MultilingualTokenizer()
         self.text_column = config.get("text_column")
+        self.label_column = config.get('label_col_name')
```

Comment:
Minor comment: why not keeping name conistency? (ie naming it self.label_col_name)

Reply:
Indeed, fixed in [0028902](https://github.com/dataiku/dss-plugin-ml-assisted-labeling/pull/64/commits/00289029a358001895c507b9c1f5f439f04f187f)